src/pilot/assign_role.py

```diff
@@ -31,6 +31,7 @@
 from dracclient import utils
 from dracclient import client
 from dracclient.constants import POWER_OFF
+from dracclient.resources import raid
```

Comment:
Once the change has been made below to go through the client object, I don't think this import will be needed any more.

Reply:
copy

Reply:
done